hawkular-alerts-api/src/main/java/org/hawkular/alerts/api/json/JacksonDeserializer.java

```diff
@@ -377,6 +467,30 @@ public static Condition deserializeCondition(JsonNode node) throws JsonProcessin
         return map;
     }
 
+    public static class ConditionException extends JsonProcessingException {
```

Comment:
Not sure if the RestEasy will handle the @ApplicationException, but anyway, adding the annotation is harmless and it can save long stack of exceptions.
I will add it.